jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -74,5 +46,12 @@ static void disableHttpLogging() {
     java.util.logging.Logger.getLogger(HttpTransport.class.getName()).setLevel(Level.OFF);
   }
 
+  static boolean isWarContainerization(Project project, RawConfiguration rawConfiguration) {
+    if (!rawConfiguration.getPackagingOverride().isPresent()) {
+      return GradleProjectProperties.getWarTask(project) != null;
+    }
+    return "war".equals(rawConfiguration.getPackagingOverride().get());
```

Comment:
So we can assume at this point packagingOverride has already been validated?

Reply:
I hope. Or it is technically possible to validate the value here. No decision yet. This PR doesn't do any validation (just the TODO for it as you saw above). But for now, any value than "war" will be considered the "flat" packaging (no error). It is not clear where is the best place to validate raw config values. I see some of them are done here and there, and as such, I haven't added validation yet. I have been thinking it'd be nice to have a global config validation phase in a pipeline, but we are not there yet.